### PR TITLE
fix(admin): adjust onClick of create webhook button in empty state

### PR DESCRIPTION
**Pull request by Tonours**
### What does it do?

Adjust onClick props of "create new webhook" button of Settings -> Webhook page to make it works with `goTo` helper.

### Why is it needed?

It's needed because otherwise the button doesn't redirect the user to the webhook creation page.

### How to test it?

1. Navigate to "Settings" -> "Webhook"
2. Ensure to have no webhooks in your database
3. Click on "create new webhook" button diplayed in the content area of the page
4. Notice that you are now redirected to the creation page of a webhook

### Related issue(s)/PR(s)

Related to this issue #19257

### CODE REVIEW

File: packages/core/admin/admin/src/pages/Settings/pages/Webhooks/ListPage.tsx [hunk 1]
```diff
@@ -383,7 +383,7 @@ const ListPage = () => {
                 <Button
                   variant="secondary"
                   startIcon={<Plus />}
-                  onClick={() => (canCreate ? goTo('create') : {})}
+                  onClick={canCreate ? goTo('create') : undefined}
```
Comment (simotae14) [inline packages/core/admin/admin/src/pages/Settings/pages/Webhooks/ListPage.tsx:386]:
```suggestion
                  onClick={canCreate && goTo('create')}
```

**Review by simotae14 — CHANGES_REQUESTED** [review]
It works ✨  Please try my suggestion, thanks

File: packages/core/admin/admin/src/pages/Settings/pages/Webhooks/ListPage.tsx [hunk 2]
```diff
@@ -383,7 +383,7 @@ const ListPage = () => {
                 <Button
                   variant="secondary"
                   startIcon={<Plus />}
-                  onClick={() => (canCreate ? goTo('create') : {})}
+                  onClick={canCreate && goTo('create')}
```
Comment (joshuaellis) [inline packages/core/admin/admin/src/pages/Settings/pages/Webhooks/ListPage.tsx:386]: we should just disabled the button if the user cannot create?

**Review by joshuaellis — CHANGES_REQUESTED** [review]

**Comment by joshuaellis** [comment]
Closing this PR due to inactivity.

**Pull request closed without merging**